arangodb-net-standard/ApiClientBase.cs

```diff
@@ -28,8 +28,16 @@ public ApiClientBase(IApiClientSerialization serialization)
         protected async Task<ApiErrorException> GetApiErrorException(IApiClientResponse response)
         {
             var stream = await response.Content.ReadAsStreamAsync();
-            var error = _serialization.DeserializeFromStream<ApiErrorResponse>(stream);
-            return new ApiErrorException(error);
+            try
+            {
+                var error = _serialization.DeserializeFromStream<ApiErrorResponse>(stream);
+                return new ApiErrorException(error);
+            }
+            catch (Exception e)
+            {
+                throw new SerializerException($"An error occured while Deserializing an error response from Arango. See InnerExpection for more details.", e);
```

Comment:
It's probably optimized away by the compiler but we don't need an interpolated string `$` here do we?